kafka-rest/src/test/java/io/confluent/kafkarest/unit/JsonSchemaConverterTest.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.unit;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.NumericNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import io.confluent.kafka.schemaregistry.json.JsonSchema;
+import io.confluent.kafkarest.TestUtils;
+import io.confluent.kafkarest.converters.ConversionException;
+import io.confluent.kafkarest.converters.JsonSchemaConverter;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class JsonSchemaConverterTest {
+
+  private static final String recordSchemaString =
+      "{\"properties\": {\n"
+      + "     \"null\": {\"type\": \"null\"},\n"
+      + "     \"boolean\": {\"type\": \"boolean\"},\n"
+      + "     \"number\": {\"type\": \"number\"},\n"
+      + "     \"string\": {\"type\": \"string\"},\n"
```

Comment:
FYI: JsonSchemaConverterTest does not work on my machine (even though it's passing on Jenkins!!). There's an extra comma in the end of this line that makes this an invalid JSON. The whole test fails to initialise.

Reply:
Ack.  I just switched to a stricter JSON parser, that's why